.github/workflows/nightly.yaml

```diff
@@ -0,0 +1,159 @@
+name: Nightly build
+
+on:
+  push:
+    branches: master
+  schedule:
+    - cron: "0 1 * * *"
+
+jobs:
+
+  dockerfile-lint:
+
+    if: github.repository_owner == 'Autoware-AI'
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+      fail-fast: false
+
+    container:
+      image: hadolint/hadolint:latest-debian
+
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Linting
+        run: |
+          hadolint generic/Dockerfile
+          hadolint generic/Dockerfile.legacy.catkin
+          hadolint generic/Dockerfile.legacy.colcon
+          hadolint generic/Dockerfile.base
+          hadolint generic/Dockerfile.cuda.kinetic
+          hadolint generic/Dockerfile.cuda.melodic
+          hadolint crossbuild/Dockerfile.kinetic-crossbuild
+          hadolint crossbuild/Dockerfile.kinetic-crossbuild-driveworks
+          hadolint crossbuild/Dockerfile.melodic-crossbuild
+          hadolint crossbuild/Dockerfile.melodic-crossbuild-driveworks
```

Comment:
The `dokerfile-lint` job doesn't really need to be run nightly. All MRs and `master` builds will run linting so it isn't necessary.

Reply:
Removed